From a4d2ea54a83a55999845b2f36d3f2664a4ef1f87 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Wed, 1 Jul 2015 22:23:53 +0300
Subject: [PATCH] Make EU::MakeMaker honour MANnEXT settings in generated
 manpage headers

This was inherited from early changes by Brendan O'Dea, previously
(accidentally) in debian/prefix_changes.diff and before that in
debian/extutils_hacks.diff

Patch-Name: debian/makemaker-manext.diff
Bug-Debian: https://bugs.debian.org/247370

---
 cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm
index 433a8dd..22e9e7b 100644
--- a/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm
+++ b/cpan/ExtUtils-MakeMaker/lib/ExtUtils/MM_Any.pm
@@ -1079,8 +1079,8 @@ END
     my @man_cmds;
     foreach my $section (qw(1 3)) {
         my $pods = $self->{"MAN${section}PODS"};
-        my $p2m = sprintf <<'CMD', $section, $] > 5.008 ? " -u" : "";
-	$(NOECHO) $(POD2MAN) --section=%s --perm_rw=$(PERM_RW)%s
+        my $p2m = sprintf <<CMD, $] > 5.008 ? " -u" : "";
+	\$(NOECHO) \$(POD2MAN) --section=\$(MAN${section}EXT) --perm_rw=\$(PERM_RW)%s
 CMD
         push @man_cmds, $self->split_command($p2m, map {($_,$pods->{$_})} sort keys %$pods);
     }
